src/styles/components/_dropdown.scss

```diff
@@ -0,0 +1,28 @@
+@use "styles/abstracts" as *;
+
+.c-menu {
```

Comment:
Persoonlijk zou ik hier een eigen `rootClassName` opgeven en deze gebruiken voor de styling in deze context. e.g. `c-archief-menu` / `c-dropdown__menu--default`. `c-menu` gaat down-the-line voor verwarring zorgen vermoed ik.

Reply:
[pr ](https://github.com/viaacode/react-components/pull/13) react-components pr voor deze change